src/app/cli/src/cli_parser.rs

```diff
@@ -1334,14 +1334,32 @@ fn value_parse_dataset_name(s: &str) -> Result<DatasetName, String> {
 }
 
 fn value_parse_dataset_ref_local(s: &str) -> Result<DatasetRef, String> {
-    match DatasetRef::try_from(s) {
+    let dataset_ref_pattern = DatasetRefPattern {
+        pattern: s.to_string(),
+        ..DatasetRefPattern::default()
+    };
+    let dataset_ref = s.to_string().replace(dataset_ref_pattern.wildcard, "ref");
+    match DatasetRef::try_from(dataset_ref.as_str()) {
         Ok(v) => Ok(v),
         Err(_) => {
             Err("Local reference should be in form: `did:odf:...` or `my.dataset.id`".to_string())
         }
     }
 }
 
+fn value_parse_dataset_ref_pattern_local(s: &str) -> Result<String, String> {
+    let arg = s.to_string();
+    if arg.contains(DatasetRefPattern::default().wildcard) {
+        return Ok(arg);
+    }
```

Comment:
I don't think it's enough. You can have a complete b*t with wildcard symbol, and it will pass.
I think we should implement another grammar in `src/domain/opendatafabric/src/identity/grammar.rs`